### ISSUE TITLE
There may be a problem with modestly deep recursion #5470

### ISSUE THREAD

**Issue by StoneCypher** [swc-project/swc#6038]
### Describe the bug

Hi.  There's an extremely clear and well described bug with reproductions in #5470.

This bug also exists in many other bugs by other people, including many other reproduction cases, and it has been declared fixed twice.

Several core authors ran my reproduction case, agreed with it, diagnosed it, and located the problem code.

Unfortunately, you run a bot that closes issues after a month, so @kdy1 asked me to open a new ticket with a reproduction case, instead of just retaining the one there already is.

This is that ticket.

The reproduction case is in incorrectly closed ticket #5470.

I'm sorry that I sound so angry, but this is a significant bug that has already been scuttled half a dozen times, and I don't like that it's being scuttled again when several people already honored it, because each time I reopen this, I have to re-justify it.

### Input code

```typescript
The input code is in incorrectly closed ticket #5470.
```

### Config

```json
The config is in incorrectly closed ticket #5470.  It is the Deno config.
```

### Expected behavior

The expected behavior is to keep valid tickets about medium severity parser bugs  with good reproductions in real world high profile open source projects open, until they have been addressed.

### Actual behavior

The ticket gets closed and you're told to start over from scratch.  See #5470 .

### Version

This appears to go back more than three years, but I cannot say how far.  It is in current.

### Additional context

I'm really sorry that I sound so angry

I hope that you folks reconsider swc-bot.  This is the sixth time it has incorrectly buried this bug.  I think bots closing issue tickets is very bad practice, personally.

Edit: removed gendered language "guys" in favor of "folks."  Sorry, very angry.  I'm trying to do better.

**Comment by StoneCypher** [comment]
you know what?  forget it.

**Comment by kdy1** [comment]
> Describe the bug
> Hi. There's an extremely clear and well described bug with reproductions in #5470.
> 
> This bug also exists in many other bugs by other people, including many other reproduction cases, and it has been declared fixed twice.
> 
> Several core authors ran my reproduction case, agreed with it, diagnosed it, and located the problem code.
> 
> Unfortunately, you run a bot that closes issues after a month, so @kdy1 asked me to open a new ticket with a reproduction case, instead of just retaining the one there already is.
> 
> This is that ticket.
> 
> The reproduction case is in incorrectly closed ticket #5470.
> 
> I'm sorry that I sound so angry, but this is a significant bug that has already been scuttled half a dozen times, and I don't like that it's being scuttled again when several people already honored it, because each time I reopen this, I have to re-justify it.
> 
> Input code
> The input code is in incorrectly closed ticket #5470.
> Config
> The config is in incorrectly closed ticket #5470.  It is the Deno config.
> Playground link
> No response
> 
> Expected behavior
> The expected behavior is to keep valid tickets about medium severity parser bugs with good reproductions in real world high profile open source projects open, until they have been addressed.
> 
> Actual behavior
> The ticket gets closed and you're told to start over from scratch. See #5470 .
> 
> Version
> This appears to go back more than three years, but I cannot say how far. It is in current.
> 
> Additional context
> I'm really sorry that I sound so angry
> 
> I hope that you guys reconsider swc-bot. This is the sixth time it has incorrectly buried this bug. I think bots closing issue tickets is very bad practice, personally.

I'll block you from the org
